icu4c/source/configure.ac

```diff
@@ -100,6 +100,16 @@ UCONFIG_CPPFLAGS=""
 # such as -std
 UCONFIG_CFLAGS=""
 
+# Check whether to install icu-config
+AC_ARG_ENABLE([icu-config],
+    AS_HELP_STRING([--enable-icu-config], [install icu-config]),
+        [case "${enableval}" in
+            yes) enable_icu_config=true ;;
+            no) enable_icu_config=false ;;
+            *) AC_MSG_ERROR([bad value '${enableval}' for --enable-icu-config]) ;;
+        esac], [enable_icu_config=false])
```

Comment:
sorry, one thing- can we make this default to true for now and announce that it will be false in the future?

Reply:
Not a problem. If I change the default case to `true`, icu-config will be installed by default and users won’t need to specify any `configure` flags for this.